packages/devtools_app/lib/src/theme.dart

```diff
@@ -10,13 +10,18 @@ import 'config_specific/ide_theme/ide_theme.dart';
 import 'ui/theme.dart';
 
 const DEFAULT_FONT_SIZE = 14.0;
+var fontSizeFactor = 1.0;
```

Comment:
this data should not be stored as a top level field like this as that could easily cause problems due to being fragile.
For example, now the themeFor method has a surprising side effect.
It is better to use the font sizes froom the existing textTheme part of a theme.

Reply:
I was hoping to use the theme to find the font size, but it looked like this would mean passing it through a lot of logic for generating inspector rows. Is there any way around doing that?